src/renderer/mixins/currency.js

```diff
@@ -85,7 +85,7 @@ export default {
         config.currency = cryptoPlaceholder
       }
 
-      const formatted = this.$n(value.toString(), config)
+      const formatted = this.$n(value.toString(), config).replace(/\s/g, ' ')
```

Comment:
This whitespace is required to keep the symbol/code on the same line as the value. To avoid this in tight spaces:

```
$ 
1000
```

Reply:
Interesting! I can revert the respective commits. With the current implementation there are two (white-)spaces between the currency symbol and the value.